### ISSUE TITLE
Typo in part 10 of finger tutorial

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6508]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#6508|
|Type|defect|
|Created|2013-05-07 13:30:32Z|

[doc/core/howto/tutorial/library.xhtml](doc/core/howto/tutorial/library.xhtml) uses this command to start the finger tutorial:

```
twisted -ny simple-finger.tac
```

This results in an error; `twisted` should be `twistd`.

Attachments:

* [lroot-ticket6508-01.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/51c/51cf3c3d5f7680a614d75b093b90baca331f7c81/1d4408c1d986b9d8cc8e34dbcdd24307d93b9221.patch) (433 bytes) - added by roguelynn on 2013-05-28 14:08:07Z - Initial patch - Fixed typo in tutorial docs

<details><summary>Searchable metadata</summary>

```
trac-id__6508 6508
type__defect defect
reporter__thijs thijs
priority__low low
milestone__ 
branch__ 
branch_author__roguelynn roguelynn
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy_documentation easy documentation
time__1367933432000000 1367933432000000
changetime__1370281095000000 1370281095000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch, I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [38656]) Apply lroot-ticket6508-01.patch: Fix a typo in the twistd command used in the finger tutorial.

Author: roguelynn
Reviewer: thijs
Fixes: [#6508](https://github.com/twisted/twisted/issues/6508)
